service/raft/raft_group0.cc

```diff
@@ -434,13 +434,35 @@ future<> raft_group0::join_group0(std::vector<gms::inet_address> seeds, bool as_
     group0_log.info("server {} joined group 0 with group id {}", my_id, group0_id);
 }
 
+struct group0_members {
```

Comment:
The commit message is outdated, now we're retrying when an address is missing.

Reply:
Fixed (no struct group0_members)